lib/build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
     implementation "com.squareup.okhttp3:okhttp:$versions.okhttp"
 
     testImplementation 'org.assertj:assertj-core:3.19.0'
+    testImplementation "com.google.truth:truth:1.1.3"
```

Comment:
Discussed offline but I would hold off on adding Truth while we are still using AssertJ. Both are very similar so I suggest using one or the other. If we want to use Truth, it should be straightforward to do a mass find/replace of imports followed by a few assertion tweaks wherever things may fail to compile. But I would highly recommend doing this in a separate PR.